### ISSUE TITLE
List item does not have a <ul>, <ol> parent element without a role

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1970]
**Source**

LF

**Feedback**

I installed the Deque Firefox extension and it’s showing this as a ‘serious’ issue. Do we need to add ‘list’ to the ul for the items they mention? Some of them are for the skip nav.

> List item does not have a <ul>, <ol> parent element without a role, or a role=“list”

Response from @aschmoe: 

> https://webaim.org/techniques/skipnav/#multiple
> Also maybe we should just have a singular skiplink to content instead
